docs/examples/ShaderExample/src/ShaderExample.ls

```diff
@@ -0,0 +1,170 @@
+"varying vec4 v_color0;                                                     \n" +
+"void main()                                                                \n" +
+"{                                                                          \n" +
+"    vec4 c = texture2D(u_texture, v_texcoord0);                            \n" +
+"    float gray = c.r + c.g + c.b / 3.0;                                    \n" +
+"    gl_FragColor =  vec4(vec3(gray), c.a) * v_color0;                      \n" +
+"}";
+
+        private var grayscaleShader:Shader;
+        private var invertShader:Shader;
+        private var brightnessShader:Shader;
+
+        private var time:TimeManager;
+
+        override public function run():void
+        {
+            stage.scaleMode = StageScaleMode.LETTERBOX;
+            stage.color = 0x888888;
+
+            // Load the shaders from string literals
+            grayscaleShader = new Shader();
+            grayscaleShader.load(vertexShader, grayscaleFragmentShader);
+            grayscaleShader.onBind += OnGrayscaleShaderBind;
+
+            invertShader = new Shader();
+            invertShader.load(vertexShader, invertFragmentShader);
+            invertShader.onBind += OnInvertShaderBind;
+
+            // Load this one from assets
+            brightnessShader = new Shader();
+            brightnessShader.loadFromAssets("assets/pulse.vert", "assets/pulse.frag");
+            brightnessShader.onBind += OnBrightnessShaderBind;
+
+            var square = Texture.fromAsset("assets/square.png");
+
+            // Create 4 quads, each with different shader
+
+            // The first one uses the default shader
+            var quad = new Image(square);
+            quad.x = stage.stageWidth / 4;
+            quad.y = stage.stageHeight / 4;
+            quad.width = 100;
+            quad.height = 100;
+            quad.center();
+            stage.addChild(quad);
+
+            quad = new Image(square);
+            quad.x = stage.stageWidth / 4 * 3;
+            quad.y = stage.stageHeight / 4;
+            quad.width = 100;
+            quad.height = 100;
+            quad.center();
+            quad.shader = grayscaleShader;
+            stage.addChild(quad);
+
+            quad = new Image(square);
+            quad.x = stage.stageWidth / 4;
+            quad.y = stage.stageHeight / 4 * 3;
+            quad.width = 100;
+            quad.height = 100;
+            quad.center();
+            quad.shader = brightnessShader;
+            stage.addChild(quad);
+
+            quad = new Image(square);
+            quad.x = stage.stageWidth / 4 * 3;
+            quad.y = stage.stageHeight / 4 * 3;
+            quad.width = 100;
+            quad.height = 100;
+            quad.center();
+            quad.shader = invertShader;
+            stage.addChild(quad);
+
+            // For FPS independent rotation
+            time = new TimeManager();
```

Comment:
It is, you have to Inject it. See https://github.com/LoomSDK/LoomSDK/blob/master/docs/examples/Poly/src/Gameplay/PolyLevel.ls#L38